### DISCUSSION TITLE
Sort by not working in latest version

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#155] [1. Help] [answered]
Hi,

Sort by functionality isn't working on latest version of breeze. I am getting attached error in console when I select any option of sorting.

![image](https://github.com/breezefront/community/assets/132253509/92b6a513-692f-416f-a285-1c6591044728)

**Comment by vovayatsyuk** [comment, marked as answer]
Looks like it's an error from the Swissup_Ajaxlayerednavigation module. Please update it to 1.4.9 (Just released)

**Reply by dheerajKsharma** [reply]
Thanks! Worked like a charm :)
